release notes/v0.37.0.md

```diff
@@ -0,0 +1,71 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring, but it's just because we are preparing some incredible things in upcoming releases!
+
+## New Features!
+
+### Added support for local files to --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+Added file hook for a logger with the possibility to define a minimal reporting level.
+
+The feature speaks for itself with examples of usage:
+
+```sh
+k6 run --log-output file=./k6.log --logformat json ./examples/stages.js
+```
+
+with a defined log level:
+
+```sh
+k6 run --log-output file=./k6.log,level=info --logformat json ./examples/stages.js
+```
+
+Thanks, @[alyakimenko](https://github.com/alyakimenko) for the contribution!
+
+**Docs**: [Using file output](https://k6.io/docs/using-k6/options/#file)
+
+### Added experimental basic event loops ([#882](https://github.com/grafana/k6/issues/882))
+
+We added the basic functionality of the JavaScript event loops ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step that is limited by a `setTimeout` method as an example. We advise avoiding using it in production code. But for sure, we'll continue to work on extending that feature in the next k6's releases.
+
+If you are curious and can't wait to try it, there is a particular `k6/experimental` module.
+
+```js
+import {setTimeout} from "k6/experimental";
+
+export default () => {
+    setTimeout(()=> {console.log("say hi to event loops!")}, 200);
+}
+```
+
+## Extensions
+
+`v0.37.0` finalizes ([#2376](https://github.com/grafana/k6/pull/2376) and [#2384](https://github.com/grafana/k6/pull/2384)) the switching of our internal modules to [a new Go/JavaScript module API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension). The k6's GRPC module was refactored in order to use the new k6's modules API ([#2365](https://github.com/grafana/k6/pull/2365)) and marked headers' parameter as deprecated ([#2370](https://github.com/grafana/k6/pull/2370))
```

Comment:
```suggestion
`v0.37.0` finalizes ([#2376](https://github.com/grafana/k6/pull/2376) and [#2384](https://github.com/grafana/k6/pull/2384)) the switching of our internal modules to [a new Go/JavaScript module API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension).
```
It could make sense to move this part into a dedicated point in the `Enhancements and UX improvements` section. Especially the `metadata` part is not related to the new API nor Extensions.

Reply:
Partly agree and to be honest, did that in 4c17e69c, the GRPC module refactoring is related to the section, so will keep it here.